test/conversion_test.go

```diff
@@ -746,10 +745,9 @@ metadata:
   namespace: %s
 spec:
   taskRunTemplate:
-  timeouts: 
+  timeouts:
     pipeline: 1h
   pipelineRef:
-    kind: Pipeline
```

Comment:
The `pipelineRef` does not have a `kind` field.

https://github.com/tektoncd/pipeline/blob/d7145450d6d87d64261abc08798876a41f607f68/pkg/apis/pipeline/v1beta1/pipelineref_types.go#L19-L37

https://github.com/tektoncd/pipeline/blob/d7145450d6d87d64261abc08798876a41f607f68/pkg/apis/pipeline/v1/pipelineref_types.go#L19-L32